Apps/AdminWebClient/src/ClientApp/src/views/Stats.vue

```diff
@@ -46,10 +46,47 @@
 import { Component, Vue } from "vue-property-decorator";
 import { SERVICE_IDENTIFIER } from "@/plugins/inversify";
 import container from "@/plugins/inversify.config";
+import { ICsvExportService } from "@/services/interfaces";
 @Component
 export default class StatsView extends Vue {
-    private downloadUserInfoCSV() {}
-    private downloadUserNotesCSV() {}
-    private downloadUserCommentsCSV() {}
+    private mounted() {
+        this.csvExportService = container.get(
+            SERVICE_IDENTIFIER.CsvExportService
+        );
+    }
+    private csvExportService!: ICsvExportService;
+    private downloadUserInfoCSV() {
+        this.csvExportService
+            .getUserProfiles()
+            .then((csvData: any) => {
```

Comment:
no type for the csv data?

Reply:
not sure if it is a Stream or not, as I can't debug this due to real service is not ready yet. We can call for a blocker then.